### ISSUE TITLE
after upgrade react-native 0.61, fast refresh feature is unable on ios devices using xcode.

### ISSUE THREAD

**Issue by jihyeonjeong11** [facebook/react-native#26952]
React Native version:

System:
    OS: macOS Mojave 10.14.5
    CPU: (4) x64 Intel(R) Core(TM) i5-6600 CPU @ 3.30GHz
    Memory: 1.76 GB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 12.7.0 - /usr/local/bin/node
    Yarn: 1.19.1 - /usr/local/bin/yarn
    npm: 6.10.0 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.1, DriverKit 19.0, macOS 10.15, tvOS 13.0, watchOS 6.0
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.5900203
    Xcode: 11.1/11A1027 - /usr/bin/xcodebuild
  npmPackages:
    @react-native-community/cli: ^2.9.0 => 2.9.0 
    react: 16.8.6 => 16.8.6 
    react-native: 0.61.1 => 0.61.1 

## Steps To Reproduce

1. create a RN project.
2. start project with xcode and any ios device

Describe what you expected to happen:

fast refresh is there on developer menu.

Snack, code example, screenshot, or link to a repository:

![Image from iOS (1)](https://user-images.githubusercontent.com/44692923/67261556-2d35e500-f4dc-11e9-8273-5c32cd71407a.png)

Also, mounting and reloading on ios device is very slow.

**Comment by myktra** [comment]
I have run into this issue as well... after working the first few builds, my app would then take several seconds to mount/transition off the launch screen and fast refresh was disabled. Tried killing the packager, restarting Xcode, clearing the npm cache/node_modules, resetting and reinstalling watchman. Ultimately I had to give up and restart my entire machine... and that fixed it.

For me: macOS 10.14.5, Node 10.16, Xcode 11.1, RN 0.61.2.
